frontend/src/v4/routes/viewerGui/components/groups/components/groupDetails/groupDetails.component.tsx

```diff
@@ -87,6 +88,14 @@ export class GroupDetails extends PureComponent<IProps, IState> {
 		return this.props.editingGroup;
 	}
 
+	get objectsCount() {
+		if (!this.isNewGroup) {
```

Comment:
This means that if I go to edit an existing group the mesh count is stuck at the riginal number. So if I edit the selected meshes this number is wrong
![image](https://github.com/3drepo/3drepo.io/assets/62481250/876878e2-1496-44f7-84e2-ba8e54543718)

Reply:
That seems correct to me tbh. When you have a new group and save, the number of objects will be the same as the selected objects, so you can see the number changing as you select/deselect some.
Opposite to that, already existing groups shouldn't update the number until you hit save as the number in the groupsCard should indicate the total number of objects belonging to the group rather than then ones currently selected in the viewer.
@sanmont3drepo what's your opinion about this?

Reply:
If it was already working this way we can ignore it and create a new issue @The-Daniel